src/foundry/client/ui/forms.d.ts

```diff
@@ -17,31 +17,38 @@ declare class FormDataExtended extends FormData {
    * @param dtypes  - A mapping of data types for form fields
    *                  (default: `{}`)
    */
+  constructor(form: HTMLFormElement);
```

Comment:
Do we really need an overload here or would it be better to just make the second parameter optional?

Reply:
TIL you *can* do that with an anonymous de-structured type, so, no it's not needed and I'll send this up in a second as an amended commit.